custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js

```diff
@@ -104,10 +104,28 @@ function MapOrSectorController($location, storageService, locationsService) {
                         var location = locations[0];
                         $location.search('location_name', location.name);
                         $location.search('location_id', location.location_id);
-
                         storageService.setKey('search', $location.search());
                         if (location.location_type_name === 'awc') {
-                            $location.path('awc_reports');
+                            var awcReportPath = 'awc_reports';
+                            // Routes for various tabs on PS page
+                            var tabRoutes = {
+                                'maternal_child': '/maternal_child',
+                                'maternal_and_child': '/maternal_child',
+                                'demographics': '/demographics',
+                                'awc_infrastructure': '/awc_infrastructure',
+                                'icds_cas_reach': '/pse',
+                            };
+                            for (var tab in tabRoutes) {
+                                if ($location.path().indexOf(tab) !== -1) {
+                                    awcReportPath += tabRoutes[tab];
+                                    break;    
+                                }
+                            }
+                            // PSE is the deafult tab
+                            if (awcReportPath === 'awc_reports') {
```

Comment:
I don't think you need this if statement do you? if the default is `'awc_reports'` then [this line](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/static/js/icds_dashboard_utils.js#L118-L120) will handle it (this is basically the same comment I already made)